docs/src/main/sphinx/connector/metastores.md

```diff
@@ -183,25 +183,23 @@ properties:
 * - `hive.metastore.thrift.txn-lock-max-wait`
   - Maximum time to wait to acquire hive transaction lock.
   - `10m`
-* - ``hive.metastore.http.client.bearer-token``
-  - Bearer token used to authenticate with the metastore service
-    when https transport mode is used. This must not be set when
-    using http url.
+* - `hive.metastore.http.client.bearer-token`
+  - Bearer token to use for authentication with the metastore service when HTTPS
+    transport mode is used. This must not be set with an HTTP URL.
   -
-* - ``hive.metastore.http.client.additional-headers``
-  - Additional headers which can be sent to the metastore service
-    http thrift requests when using the http transport mode. These
-    headers must be comma-separated and delimited using ``:``. E.g
-    header1:value1,header2:value2 sends two headers header1 and
-    header2 with their values as value1 and value2 respectively.
-    If you need to use a comma(``,``) or colon(``:``) in a header name
-    or value, escape it using a backslash (``\``).
+* - `hive.metastore.http.client.additional-headers`
+  - Additional headers to send with metastore service Thrift requests when using
+    the HTTP client transport mode. These headers must be comma-separated and
+    delimited using `:`. For example, `header1:value1,header2:value2` sends two
+    headers `header1` and `header2` with the values as `value1` and `value2`.
+    Escape comma (`,`) or colon(`:`) characters in a header name or value with a
+    backslash (`\`).
   -
-* - ``hive.metastore.http.client.authentication.type``
-  - The authentication type to be used for the http metastore client.
-    Currently, the only supported type is ``BEARER``. When set to ``BEARER``
-    the token configured in ``hive.metastore.http.client.bearer-token``
-    is used to authenticate the client to the http metastore service.
+* - `hive.metastore.http.client.authentication.type`
```

Comment:
I feel it would be better to change the orders like below:
* hive.metastore.http.client.authentication.type
* hive.metastore.http.client.bearer-token
* hive.metastore.http.client.additional-headers

Reply:
Done .. also pulled out into separate table with intro sentence to avoid duplication of explaining the HTTP mode